lib/Solver/FastCexSolver.cpp

```diff
@@ -288,7 +290,7 @@ class ValueRange {
     if (m_min < smallest && m_max >= smallest) {
       return smallest - 1;
     } else {
-      return ints::sext(m_max, 64, bits);
+      return llvm::APInt(m_max, bits, true).getSExtValue();
```

Comment:
`m_max` and `bits` are in the wrong order here.

Reply:
Great catch! Fixed.